### PR TITLE
[SMTC-1406] Create toolbar for feedback activity

### PR DESCRIPTION
**Pull request by ivamluz**

### CODE REVIEW

File: app/src/main/res/layout/activity_feedback.xml [hunk 1]
```diff
@@ -13,4 +13,41 @@
...
         app:dlfvCameraOrnamentScaleType="center"/>
+
+    <android.support.v7.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize"
+        style="@style/AppTheme.Toolbar"
+        android:elevation="@dimen/toolbar_elevation">
+
+        <RelativeLayout
```
Comment (jeanpimentel) [inline app/src/main/res/layout/activity_feedback.xml:24]: Why implement these views manually?

Why not
```
setDisplayHomeAsUpEnabled(true);
setDisplayShowHomeEnabled(true); // for arrow

// and

onCreateOptionsMenu // for options
```

Comment (ivamluz) [reply app/src/main/res/layout/activity_feedback.xml:24]: If I enable the home button, it splits the toolbar in two. Then, in order to centralize the title, I would have to use a negative margin (to ignore the size of the area dedicated to the home button). Do you think it's better?

Comment (jeanpimentel) [reply app/src/main/res/layout/activity_feedback.xml:24]: I thought we would keep to the left.
But no problems, this makes sense.

**Review by jeanpimentel — APPROVED** [review]

File: app/src/main/res/layout/activity_feedback.xml (hunk 1 above)
Comment (ivamluz) [reply app/src/main/res/layout/activity_feedback.xml:24]: selection_241

**Pull request merged**